site/src/pages/[shortname]/blocks.page.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { NextPage, GetServerSideProps } from "next";
+
```

Comment:
Instead of creating 2 new files, I think it's better we leverage [Rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites) in Nextjs to point `/[shortname]/blocks` and `/[shortname]/schemas` routes to `/shortname` route. I think we already use that for [docs](https://github.com/blockprotocol/blockprotocol/blob/main/site/next.config.js#L22)

Reply:
Thank you Teenoh! This has now been implemented.